msteams-platform/concepts/outgoingwebhook.md

```diff
@@ -1,38 +1,43 @@
 ---
-title: Custom bots
-description: Describes how to create and use custom bots in Microsoft Teams
+title: Outgoing webhooks
+description: Describes how to create and use outgoing webhooks in Microsoft Teams
 keywords: teams bots custom
 ---
 
-# Custom bots in Microsoft Teams
+# Outgoing webhooks in Microsoft Teams
 
-If you've integrated with outgoing webhooks or slash commands in other chat platforms, you can now easily bring those integrations over to Microsoft Teams via Custom Bots.  Custom bots are an easy way of extending your team with Teams interactivity without having to go through the full process of creating a bot via the Microsoft Bot Framework.  You can use them for custom workflows and commands, such as kicking off a build or checking the latest set of livesite issues. With custom bots, too, you have effective way of ensuring your service is accessible only by authorized users, as the shared secret used by your custom bot will only be scoped to the team in which it has been added.
+If you've worked with outgoing webhooks or slash commands in other chat platforms, you can now bring what you have developed over to Microsoft Teams via outgoing webhooks.  Outgoing webhooks are an easy way of extending your team without having to go through the full process of creating a bot via the Microsoft Bot Framework. You can use them for custom workflows and commands such as kicking off a build or checking the latest set of livesite issues.
 
-## Creating a custom bot
+You also have an effective way of ensuring that your service is accessible only by authorized users, as the security token used by your outgoing webhook will only be scoped to the team in which it has been added.
 
-To create a custom bot, click View Team and then navigate to the Bots tab.
+## Creating an outgoing webhook
 
-![View team](~/assets/images/tab_view_team.png)
+To create an outgoing webhook, click **Manage team** and then navigate to the **Bots** tab.
 
-Click on the Create a custom bot link at the bottom of the page.
+![View team](~/assets/images/ManageTeam.png)
+
+Click on the **Create a outgoing webhook** link at the bottom of the page.
  
-![Create a custom bot entry point](~/assets/images/sideloadentrypoint.png)
+![Create a outgoing webhook entry point](~/assets/images/createwebhook.png)
+
+In the **Create an outgoing webhook** dialog, you can configure how your bot appears in channels:
```

Comment:
"configure how your bot appears in channels" s/b "configure how your outgoing webhook appears in channels"

Reply:
Done